pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1501,6 +1524,14 @@ type BackupScheduleSpec struct {
 	// ImagePullSecrets is an optional list of references to secrets in the same namespace to use for pulling any of the images.
 	// +optional
 	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
+
+	// Additional annotations for backup schedule pods
+	// +optional
+	Annotations map[string]string `json:"annotations,omitempty"`
+
+	// Additional labels for backup schedule pods
+	// +optional
+	Labels map[string]string `json:"labels,omitempty"`
```

Comment:
Sorry, my fault, no need to add these new fields, we can just use the metadata.labels/annotations of the backupSchedule CR, and combine them with the configurations defined in the `BackupTemplate` field.